### PR TITLE
(SIMP-MAINT) Actually list pkgs in sort -V order

### PR DESCRIPTION
**Pull request by op-ct**
We note that packages are listed in `sort -V` order; now they are.

This patch also adds a `check:pkglist_lint` rake task to validate that the file is in `sort -uV` order.

### CODE REVIEW

File: build/distributions/CentOS/6/x86_64/DVD/6-simp_pkglist.txt [hunk 1]
```diff
@@ -802,19 +805,19 @@ perl-IO-Compress-Zlib
...
 perl-Mozilla-LDAP
-perl-Net-Telnet
 perl-NetAddr-IP
+perl-Net-Telnet
 perl-Params-Validate
 perl-Pod-Escapes
 perl-Pod-Simple
-perl-Readonly
-perl-Readonly-XS
+# perl-Readonly
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/DVD/6-simp_pkglist.txt:813]: Why did you comment out this and the subsequent package?  Were you able to build and ISO and run packer tests without this successfully?

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 2]
```diff
@@ -130,23 +132,22 @@ brltty
...
 bzip2
 bzip2-libs
-c-ares
-ca-certificates
 cairo
 cairo-gobject
 cal10n
-# caribou* only in rhel-server distribution ISO
 caribou
+# caribou* only in rhel-server distribution ISO
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt:139]: With the sorting, comments are now below the related packages.  Seems odd.

Comment (trevor-vaughan) [reply build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt:139]: Yeah, that's not great. We should preserve the comments.

**Review by trevor-vaughan — CHANGES_REQUESTED** [review]
See comments.

**Review by trevor-vaughan — APPROVED** [review]

**Pull request merged**